app/assets/stylesheets/pages.scss

```diff
@@ -0,0 +1,52 @@
+// Place all the styles related to the Pages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+
+
+/* General */
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+body {
+  background-image: url('catz/catz.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  font-family: 'Kosugi', sans-serif;
+}
+
+.container {
+  margin: 0 auto;
+  width: 960px;
+}
+
+/* Main */
+.main {
+  color: #000;
+  position: relative;
+  top: 100px;
+  text-align: center;
+}
+
+.main {
+  h1 {
+    font-size: 80px;
+    font-weight: 300;
+  }
+
+  p {
+    font-size: 30px;
+  }
+}
```

Comment:
So you could actually merge these two class definitions into one (one of the great things about Sass!). This would look something like this:
```scss
.main {
  color: #000;
  position: relative;
  top: 100px;
  text-align: center;

  h1 {
    font-size: 80px;
    font-weight: 300;
  }

  p {
    font-size: 30px;
  }
}
```